css/style.css

```diff
@@ -16,7 +16,7 @@ Body{
     margin: 15px;  
  }
  
- img {
+ img{
    padding-top: 10px;
    max-width: 100%; 
```

Comment:
You should try using a width of 100% instead of a max-width of 100% because the default already has a max-width of 100%